test/main_test.js

```diff
@@ -86,3 +91,14 @@ function checkExists(path) {
     `${path} should exist`
   );
 }
+
+function checkEnvInNinja(path) {
+  let read = new Promise((resolve, reject) => {
```

Comment:
I prefer to change it by outputting a success file if a given env variable ENV_TEST=1 is existed with correct value.